src/encoded/static/components/navigation/components/UserActionDropdownMenu.js

```diff
@@ -20,6 +20,30 @@ import { LoginNavItem, LogoutDropdownItem } from './LoginNavItem';
  * @property {boolean|function} active - Whether action is currently active.
  */
 
+
+ const FFAuth0Options = {
+     auth: {
+         sso: false,
+         redirect: false,
+         responseType: 'token',
+         params: {
+             scope: 'openid email',
+             prompt: 'select_account'
+         }
+     },
+     socialButtonStyle: 'big',
+     theme: {
+         logo: '/static/img/4dn_logo.svg',
+         icon: '/static/img/4dn_logo.svg',
+         primaryColor: '#009aad'
+     },
+     allowedConnections: ['github', 'google-oauth2'],
+     languageDictionary: {
+         title: 'Log In'
+     }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 5. (indent)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4484611)